### ISSUE TITLE
Wireframes Homework (Initial Draft)

### ISSUE THREAD

**Issue by SoItGoes33** [SoItGoes33/group-proj#2]
@Diyah, @Keenahn, & @LijieZhou

Here is the initial draft of the wireframes for our group project.  This is a rough overview of our layout which we will refine more after we discuss a bit more how we want to parse and display the data that we have.  If you have any questions and/or concerns about this, please let us know.  Thanks, Carol, Soumia and Susan

![finalproject_wireframe1 1](https://cloud.githubusercontent.com/assets/13411563/9646334/758e8322-5187-11e5-9129-d2a5b4d5a10c.png)

**Comment by keenahn** [comment]
Cool, so this is going to just be a single page app? Looks good :) Excited to see what kind of visualizations you come up with.

**Comment by ghost** [comment]
Yes :) we want it to flow like a story and we're super excited about it!!!
